packages/widgets/src/indexer-manager/component.tsx

```diff
@@ -39,26 +36,49 @@ export default function IndexerManagerWidget({
     },
   );
 
+  const iconStyle = { height: "7.5cqmin", width: "7.5cqmin" };
+
+  if (integrationIds.length === 0) {
+    throw new NoIntegrationSelectedError();
+  }
```

Comment:
Move to top of function

Reply:
Why? Doesn't this cause hooks issues? I remember needing to do that for the media request widgets with Meier.

Reply:
Ah, sorry. Didn't see the hook. Approved.